state-chain/pallets/cf-vaults/src/vault_activator.rs

```diff
@@ -40,12 +43,17 @@ impl<T: Config<I>, I: 'static> VaultActivator<<T::Chain as Chain>::ChainCrypto>
 					);
 				},
 			}
-		} else {
-			// No active key means we are bootstrapping the vault.
+			FirstVault::False
+		} else if ChainInitialized::<T, I>::get() {
+			// VaultStartBlockNumbers being empty means we are bootstrapping the vault.
```

Comment:
This confirms a migration is required too, the existing chains need this set to true, and arb set to false